modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLSSOForAdminLoginTestCase.java

```diff
@@ -0,0 +1,183 @@
+import org.wso2.carbon.integration.common.utils.FileManager;
+import org.wso2.carbon.integration.common.utils.exceptions.AutomationUtilException;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+import org.wso2.identity.integration.test.util.Utils;
+import org.wso2.identity.integration.test.utils.CommonConstants;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.xml.xpath.XPathExpressionException;
+
+/**
+ * This class contains test cases for SAML SSO for Admin Console.
+ */
+public class SAMLSSOForAdminLoginTestCase extends AbstractSAMLSSOTestCase {
+
+    private static final Log log = LogFactory.getLog(SAMLSSOForAdminLoginTestCase.class);
+    private ServerConfigurationManager serverConfigurationManager;
+
+    protected static final String SAML_SSO_URL = "https://localhost:9853/samlsso";
+    protected static final String MANAGEMENT_CONSOLE_LOGIN_URL = "https://localhost:9853/carbon/admin/login.jsp";
+    protected static final String MANAGEMENT_CONSOLE_HOME_URL = "https://localhost:9853/carbon/admin/index.jsp";
+
+    private SAMLConfig config;
+
+    @Factory(dataProvider = "samlConfigProvider")
+    public SAMLSSOForAdminLoginTestCase(SAMLConfig config) {
+
+        if (log.isDebugEnabled()) {
+            log.info(String.format("SAML SSO for Admin Console Test initialized for %s", SAML_SSO_URL));
+        }
+        this.config = config;
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void testInit() throws Exception {
+
+        super.init(config.getUserMode());
+        super.testInit();
+        changeISConfiguration();
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void testClear() throws Exception {
+
+        resetISConfiguration();
+        super.testClear();
+    }
+
+    @Test(alwaysRun = true, description = "Testing SSO for admin login", groups = "wso2.is")
+    public void testSSOForAdminLogin() {
+
+        try {
+            // Get to management console login page
+            HttpResponse response = Utils.sendGetRequest(MANAGEMENT_CONSOLE_LOGIN_URL, USER_AGENT, httpClient);
+            Assert.assertEquals(response.getStatusLine().getStatusCode(), 200,
+                    "User may have already logged in");
+
+            // Extract saml request from the response and send the post request to samlsso endpoint
+            String samlRequest = Utils.extractDataFromResponseForManagementConsoleRequests(response, "value", 1);
+            response = sendSAMLMessage(SAML_SSO_URL, CommonConstants.SAML_REQUEST_PARAM, samlRequest, config);
+            EntityUtils.consume(response.getEntity());
+            response = Utils.sendRedirectRequest(response, USER_AGENT, ACS_URL, SAML_SSO_URL, httpClient);
+            Assert.assertEquals(response.getStatusLine().getStatusCode(), 200,
+                    "SAML SSO Login failed for " + config);
+
+            // check whether the user is logged in
```

Comment:
Let's use sentence case for comments :)
Please fix other instances as well

Reply:
Resolved with d76ca3f77f15f91e003adb1fc0319feff665db9c